proto/rill/runtime/v1/resources.proto

```diff
@@ -153,6 +153,17 @@ message MetricsViewSpec {
     repeated FieldConditionV2 include = 3;
     repeated FieldConditionV2 exclude = 4;
   }
+  enum DefaultComparisonMode {
+    DEFAULT_COMPARISON_MODE_UNSPECIFIED = 0;
+    DEFAULT_COMPARISON_MODE_NONE = 1;
+    DEFAULT_COMPARISON_MODE_TIME = 2;
+    DEFAULT_COMPARISON_MODE_DIMENSION = 3;
+  }
```

Comment:
Just realized we can change this to `ComparisonMode` (since "default" is more like a variable than a type). So it will be `ComparisonMode default_comparison_mode = 14;`